02_bubble/madhavansu/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, bubble([]int{3, 2, -2092, -5, 9, 0, 29, 19, -2, -2, -10}))
+	fmt.Fprint(out, insertion([]int{3, 2, -2092, -5, 9, 0, 29, 19, -2, -2, -10}))
+}
+
+func bubble(arr []int) []int {
+	arrlen := len(arr)
+	for i := 0; i < arrlen-1; i++ {
+		for j := 0; j < arrlen-i-1; j++ {
+			if arr[j] > arr[j+1] {
```

Comment:
You are modifying the input.
For that reason you cannot re-use your test cases.
make a copy of `arr` first.

Reply:
Done